src/qt/locale/bitcoin_nl.ts

```diff
@@ -2513,15 +2466,15 @@ p, li { white-space: pre-wrap; }
     </message>
     <message>
         <source>Amount:</source>
-        <translation>Bedrag:</translation>
+        <translation>Brutobedrag:</translation>
```

Comment:
If this is only meant to show the amount excluded the fees, then this is correct.